docs/diagrams/ExportScheduleActivityDiagram.puml

```diff
@@ -0,0 +1,8 @@
+@startuml
+start
+:User executes command;
+
+:Save schedule;
+
+stop
+@enduml
```

Comment:
Is this activity diagram really necessary? As it is really simple and I think the developers who read the DG can understand it only by its description without the diagram.